File: Data Structures & Algorithms/is-anagram/submission-2.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = {}

        print("Print characters from S")
        for ch in s:
            count[ch] = count.get(ch, 0) + 1
            print(ch, count[ch])
        
        print("Print characters from T")
        for ch in t:
            count[ch] = count.get(ch, 0) - 1
            print(ch, count[ch])

        print(count)
        print("Print Count Values")
        for val in count.values():
            print(val)
            if val != 0:
                return False

        return True
        

File: Data Structures & Algorithms/daily-temperatures/submission-2.cs
public class Solution {
    public int[] DailyTemperatures(int[] temperatures) {
        int length = temperatures.Length;
        int[] result = new int[length];
        var stack = new Stack<int>();

        for (int i = 0; i < length; i++) {
            while (stack.Count > 0 && temperatures[i] > temperatures[stack.Peek()]) {
                int prevIndex = stack.Pop();
                result[prevIndex] = i - prevIndex;
            }

            stack.Push(i);
        }

        return result;
    }
}
